recipes/librdkafka/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+import os
+
+from conans import CMake, ConanFile, tools
+
+
+class LibrdkafkaConan(ConanFile):
+    name = "librdkafka"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/edenhill/librdkafka"
+    description = "Librdkafka is an Apache Kafka C/C++ library designed with message delivery reliability and high performance in mind."
+
+    topics = ("kafka", "librdkafka")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "zlib": [True, False],
+        "zstd": [True, False],
+        "plugins": [True, False],
+        "ssl": [True, False],
+        "sasl": [True, False],
+        "lz4": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "zlib": False,
+        "zstd": False,
+        "plugins": False,
+        "ssl": False,
+        "sasl": False,
+        "lz4": False,
+    }
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package"
```

Reply:
That was a good point because much less patching was required in order to link libraries.
And I accidentally removed target_include_directories in my patches before this.